### DISCUSSION TITLE
Transfer an organization?

### DISCUSSION THREAD

**Discussion by ehmatthes** [django-commons/membership#18] [General]
Hi! First of all, thank you for creating this. It's a fantastic idea, clearly builds on prior efforts in this area, and looks like it's off to a great start.

Is there a mechanism for transferring an organization in? I don't have a specific org in mind, but I'm thinking of all the open projects I've seen that are structured as orgs because the project consists of more than one repo.

I am not aware of support for nested orgs, but maybe teams could be used to implement support for an org? Transfer all the repos from the org, then make a team that has access to those specific repos?

**Comment by tim-schilling** [comment]
Hi Eric, thanks for reaching out. Currently I have an issue template for "Transfer project in" available at: https://github.com/django-commons/membership/issues/new/choose (new issue on the membership repo). The playbook an admin/organizer will follow can be found in the [controls repo here](https://github.com/django-commons/controls?tab=readme-ov-file#new-project-playbook)

**Reply by ehmatthes** [reply]
This all looks good. As a side note, the [ARCHITECTURE](https://matklad.github.io/2021/02/06/ARCHITECTURE.md.html) post is exactly what I was looking for this past week, so I appreciate the inclusion of that.

**Comment by tim-schilling** [comment]
I didn't have anything in mind for handling an org transferring an entire suite of projects. I think it'd be best to treat them individually since I don't expect to have sub-organizations in Django Commons. Or at least not right now. Until the org has its legs underneath it, it should approach things in the most simple, repeatable manner.

**Reply by ehmatthes** [reply]
Sounds good! Thanks for getting this off the ground!
